Pidaq/.gitignore

```diff
@@ -0,0 +1 @@
+!important vscode/tasks.json
```

Comment:
Should this be .vscode/tasks.json? I think that directory is still getting gitignored, just noticed this when following the instructions on the README

Reply:
You're correct, fixed